storybook/src/elements/nuxeo-actions-menu/nuxeo-actions-menu.stories.js

```diff
@@ -15,9 +15,9 @@ storiesOf('UI/Widgets', module).addElement('nuxeo-actions-menu', ({ knobs }) =>
     <nuxeo-actions-menu>
       ${Array.from({ length: number('# actions', 10) }).map(
         () => html`
-          <nuxeo-link-button href="javascript:void(0)" icon="nuxeo:edit" label="Dummy"></nuxeo-link-button>
+          <nuxeo-link-button href="javascript:void(0)" icon="nuxeo:edit"></nuxeo-link-button>
```

Comment:
This raises me a question: do we want to have an example with labels? (by using `label` and `show-label` properties)

Reply:
we cannot use label as is controlled by the menu, but i did add the labels back.